airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1199,11 +1185,26 @@ def run_load(self,  # pylint: disable=too-many-locals,too-many-arguments,invalid
             'PARQUET': ['autodetect', 'ignoreUnknownValues'],
             'AVRO': ['useAvroLogicalTypes'],
         }
+
         valid_configs = src_fmt_to_configs_mapping[source_format]
-        src_fmt_configs = {
-            k: v
-            for k, v in src_fmt_configs.items() if k in valid_configs
-        }
+
+        # if following fields are not specified in src_fmt_configs,
```

Comment:
That screams for extracting to common method reused in two pleaces.

Reply:
Sure I will make the change!

Reply:
@potiuk Pushed the change